src/registrar/utility/csv_export.py

```diff
@@ -44,7 +45,9 @@ def get_domain_infos(filter_condition, sort_fields):
     return domain_infos_cleaned
 
 
-def parse_row(columns, domain_info: DomainInformation, security_emails_dict=None, get_domain_managers=False):
+def parse_row(
+    columns, domain_info: DomainInformation, max_dm_count, security_emails_dict=None, get_domain_managers=False
+):
```

Comment:
**non blocking suggestion**: this is for a section not part of your changes
This part below will just cause an unhelpful 500 error to show on screen. Fine for now as hopefully this should never happen but bad practice, we should at least be logging the error before throwing errors. 

Mind adding a log there? Not
```
 if domain_info.domain is None:
        raise ValueError("Domain is none")
```